op-e2e/actions/proofs/isthmus_fork_test.go

```diff
@@ -427,4 +441,19 @@ func TestIsthmusNetworkUpgradeTransactions(gt *testing.T) {
 	latestBlock, err = ethCl.BlockByNumber(context.Background(), nil)
 	require.NoError(t, err)
 	checkRecentBlockHash(latestBlock.NumberU64()-1, latestBlock.Header().ParentHash, "post-activation")
+
+	l2Safe := sequencer.L2Safe()
+	env.RunFaultProofProgram(t, l2Safe.Number, testCfg.CheckResult, testCfg.InputParams...)
+}
+
+// verifyCodeHashMatches checks that the has of the code at the given address matches the expected code-hash.
+// It also sanity-checks that the code is not empty: we should never deploy empty contract codes.
+// Returns the contract code
+func verifyCodeHashMatches(t actionsHelpers.Testing, client *ethclient.Client, address common.Address, expectedCodeHash common.Hash) []byte {
```

Comment:
where is this used?

Reply:
It's used multiple times in `testIsthmusNetworkUpgradeTransactions`. It's copy pasted from a helper here https://github.com/ethereum-optimism/optimism/blob/d1098239e3cca7b21371c447ef0c70a2c3ee7be4/op-e2e/actions/upgrades/ecotone_fork_test.go#L34 (this was simpler than moving it somewhere to be reused).